src/Model/CalculateImages.py

```diff
@@ -88,6 +88,7 @@ def get_img(pixel_array):
     return dict_img
 
 
+
```

Comment:
Too many blank lines, should only be two here. (PEP 8)

Reply:
I didn't see that, will make those changes.